waitress/tests/test_task.py

```diff
@@ -65,6 +65,13 @@ def test_add_task(self):
         self.assertEqual(inst.queue.qsize(), 1)
         self.assertTrue(task.deferred)
 
+    def test_log_queue_depth(self):
+        task = DummyTask()
+        inst = self._makeOne()
+        inst.add_task(task)
+        inst.logger = DummyLogger()
+        self.assertEqual(len(inst.logger.logged), 1)
```

Comment:
The test needs to add a second task (so that the queue is already non-empty) in order to trigger the log entry.  Also, it might be cleaner to add the logger *before* adding the first task, and then add an assertion that no logging was done.  Then add the second task.